content/projekte/_index.md

```diff
@@ -9,16 +9,86 @@ outputs:
 
 {{< headline-icon icon="icon-documents.svg" color="blue" >}}
 
-{{< headline-brackets-h1 color="red"  >}}
-Unsere Projekte
-{{< /headline-brackets-h1  >}}
+{{< headline-brackets-h1 color="red" >}}
+Projekte
+{{< /headline-brackets-h1 >}}
 
+{{< hero-block-center >}}
+Alle Projekte, die bei Code for Germany entstehen, sind Open Source. Sie lassen sich kostenlos verwenden und weiterentwicklen.
```

Comment:
```suggestion
[Alle Projekte](projekte/alle), die bei Code for Germany entstehen, sind Open Source. Sie lassen sich kostenlos verwenden und weiterentwicklen.
```

Reply:
prerequisite for this to work: the page now called projekte/archiv is already renamed to /projekte/alle

=> see below, my comment on content/projekte/archiv/_index.md